wp-content/themes/missilethreat/assets/_scss/style.scss

```diff
@@ -17,6 +17,7 @@
 @use 'components/navigation';
 @use 'components/newsletter';
 @use 'components/page-header';
+@use 'components/pagination';
```

Comment:
I would include this just on the archive page, since you don't use it anywhere else.